apps/editors/tests/test_views.py

```diff
@@ -2312,6 +2312,14 @@ def test_version_deletion(self):
         eq_(ths.length, 2)
         assert '0.1' in ths.text()
 
+    def test_no_versions(self):
+        """A 404 should be returned if there's no version."""
+        self.version.delete()
```

Comment:
I know this is probably duplicate code but I love to verify very quickly that it worked before, e.g that it was a 200 and not already a 404 because of some weird thing.